spec/factories/order_item_factory.rb

```diff
@@ -3,7 +3,7 @@
     order
     item
     sequence(:quantity) { |n| ("#{n}".to_i+1)*2 }
-    sequence(:price) { |n| ("#{n}".to_i+1)*1.5 }
+    sequence(:price) { (item.price * quantity).to_f }
```

Comment:
This line right here!!!! This is why some model tests can pass when we run `rspec spec/models` and fail when running `rspec`. The reason is that with r`spec` the `n` in the sequence is a lot higher since more bots were created in other tests so it will adjust the price!!!